### PR TITLE
Updating Hydro routines round 2

### PR DESCRIPTION
**Pull request by aemerick**
Finalizing renaming and moving of redundant hydro routines that did not
  get fixed in prior PPM-cull PR. This moves and renames all woc_thing.F
  routines to thing.F, and removes the previous thing.F files. The previous
  thing.F files were originally named thing_dev.F before the recent PPM-cull
  PR. It appears the woc_thing.F files are more recent than the thing_dev.F
  files were, as the woc_thing.F files show fewer differences with their
  equivalents in Enzo.

  enzo_EnzoMethodHydro is retained (the routine that called the woc_thing.F routines)
  but commented out entirely to avoid confusion with the current PPM and
  SolveHydroEquations methods... This has not been removed entirely in case
  development continues here (this was originally a way to refactor and improve
  the way thigns are currently done in SolveHydroEquations).

### CODE REVIEW

**Review by jwise77 — CHANGES_REQUESTED** [review]
This all looks good! However there are a bunch of conflicts, presumably arising after the dd merge. Fix these and I'll approve.

**Comment by bwoshea** [comment]
@aemerick lots of merge conflicts here... This looks easy to fix, at least.

File: src/Enzo/enzo_EnzoMethodHydro.cpp
Comment (mabruzzo) [inline src/Enzo/enzo_EnzoMethodHydro.cpp:2]: Great, I'm really happy to see this. I found the `EnzoMethodHydro` method to be really confusing when I was first getting acquainted with the codebase.

**Review by mabruzzo — APPROVED** [review]
This all looks good to me.

**Comment by jwise77** [comment]
I will fix the conflicts today.

**Review by jwise77 — APPROVED** [review]
Fixed conflicts, and I will merge once the tests pass.

**Pull request merged**
